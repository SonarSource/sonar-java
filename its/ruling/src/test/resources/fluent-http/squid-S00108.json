{
'net.code-story:http:src/main/java/net/codestory/http/Configuration.java':[
25,
],
}
