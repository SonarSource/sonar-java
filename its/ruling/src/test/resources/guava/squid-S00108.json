{
'com.google.guava:guava:src/com/google/common/base/CharMatcher.java':[
1304,
1305,
],
'com.google.guava:guava:src/com/google/common/cache/LocalCache.java':[
2524,
2528,
],
'com.google.guava:guava:src/com/google/common/cache/Striped64.java':[
319,
],
'com.google.guava:guava:src/com/google/common/collect/Cut.java':[
96,
],
'com.google.guava:guava:src/com/google/common/collect/MapMakerInternalMap.java':[
2235,
2239,
],
'com.google.guava:guava:src/com/google/common/primitives/UnsignedBytes.java':[
349,
],
'com.google.guava:guava:src/com/google/common/reflect/ClassPath.java':[
389,
],
'com.google.guava:guava:src/com/google/common/reflect/TypeResolver.java':[
366,
],
}
