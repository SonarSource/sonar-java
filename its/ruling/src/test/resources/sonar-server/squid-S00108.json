{
'org.sonarsource.sonarqube:sonar-server:src/main/java/org/sonar/server/organization/OrganizationCreationImpl.java':[
98,
],
}
