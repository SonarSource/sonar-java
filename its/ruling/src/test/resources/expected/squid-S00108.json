{
'project:com/sun/corba/se/impl/activation/ORBD.java':[
337,
],
'project:com/sun/corba/se/impl/activation/ServerMain.java':[
103,
],
'project:com/sun/corba/se/impl/activation/ServerTableEntry.java':[
391,
441,
],
'project:com/sun/corba/se/impl/activation/ServerTool.java':[
218,
337,
338,
390,
472,
659,
743,
],
'project:com/sun/corba/se/impl/corba/RequestImpl.java':[
280,
],
'project:com/sun/corba/se/impl/corba/ServerRequestImpl.java':[
241,
],
'project:com/sun/corba/se/impl/corba/TypeCodeImpl.java':[
309,
309,
887,
887,
2283,
2284,
],
'project:com/sun/corba/se/impl/dynamicany/DynEnumImpl.java':[
90,
99,
100,
],
'project:com/sun/corba/se/impl/dynamicany/DynUnionImpl.java':[
112,
121,
130,
131,
140,
141,
150,
151,
160,
249,
282,
],
'project:com/sun/corba/se/impl/io/ObjectStreamClass.java':[
1607,
],
'project:com/sun/corba/se/impl/javax/rmi/CORBA/Util.java':[
753,
],
'project:com/sun/corba/se/impl/javax/rmi/PortableRemoteObject.java':[
347,
],
'project:com/sun/corba/se/impl/naming/cosnaming/NamingContextImpl.java':[
494,
],
'project:com/sun/corba/se/impl/naming/pcosnaming/NamingContextImpl.java':[
477,
],
'project:com/sun/corba/se/impl/naming/pcosnaming/ServantManagerImpl.java':[
128,
154,
162,
225,
239,
],
'project:com/sun/corba/se/impl/oa/poa/POAFactory.java':[
177,
],
'project:com/sun/corba/se/impl/oa/poa/POAImpl.java':[
548,
],
'project:com/sun/corba/se/impl/orb/ORBImpl.java':[
1252,
],
'project:com/sun/corba/se/impl/orb/ParserTable.java':[
160,
],
'project:com/sun/corba/se/impl/orbutil/CorbaResourceUtil.java':[
39,
],
'project:com/sun/corba/se/impl/orbutil/ORBUtility.java':[
505,
],
'project:com/sun/corba/se/impl/orbutil/ObjectStreamClassUtil_1_3.java':[
525,
],
'project:com/sun/corba/se/impl/orbutil/ObjectStreamClass_1_3_1.java':[
269,
533,
545,
573,
590,
],
'project:com/sun/corba/se/impl/protocol/giopmsgheaders/MessageBase.java':[
871,
],
'project:com/sun/corba/se/impl/transport/CorbaResponseWaitingRoomImpl.java':[
159,
],
'project:com/sun/corba/se/impl/util/JDKBridge.java':[
183,
],
'project:com/sun/corba/se/impl/util/ORBProperties.java':[
69,
],
'project:com/sun/corba/se/impl/util/Utility.java':[
229,
561,
],
'project:com/sun/imageio/plugins/jpeg/COMMarkerSegment.java':[
101,
],
'project:com/sun/imageio/plugins/jpeg/DHTMarkerSegment.java':[
212,
],
'project:com/sun/imageio/plugins/jpeg/DQTMarkerSegment.java':[
275,
],
'project:com/sun/imageio/plugins/jpeg/JFIFMarkerSegment.java':[
832,
],
'project:com/sun/imageio/plugins/jpeg/JPEGMetadata.java':[
599,
724,
],
'project:com/sun/imageio/plugins/jpeg/MarkerSegment.java':[
115,
],
'project:com/sun/imageio/plugins/jpeg/SOFMarkerSegment.java':[
255,
],
'project:com/sun/imageio/plugins/jpeg/SOSMarkerSegment.java':[
216,
],
'project:com/sun/imageio/stream/CloseableDisposerRecord.java':[
48,
],
'project:com/sun/imageio/stream/StreamCloser.java':[
75,
],
'project:com/sun/imageio/stream/StreamFinalizer.java':[
66,
],
'project:com/sun/java/swing/plaf/gtk/GTKColorChooserPanel.java':[
125,
142,
261,
270,
],
'project:com/sun/java/swing/plaf/gtk/GTKStyle.java':[
1098,
1099,
1100,
1101,
1102,
],
'project:com/sun/java/swing/plaf/gtk/Metacity.java':[
73,
],
'project:com/sun/java/swing/plaf/gtk/PangoFonts.java':[
99,
226,
],
'project:com/sun/java/swing/plaf/motif/MotifDesktopIconUI.java':[
355,
],
'project:com/sun/java/swing/plaf/motif/MotifInternalFrameTitlePane.java':[
114,
],
'project:com/sun/java/swing/plaf/windows/WindowsDesktopManager.java':[
97,
],
'project:com/sun/java/swing/plaf/windows/WindowsInternalFrameTitlePane.java':[
320,
],
'project:com/sun/java/swing/plaf/windows/WindowsTextFieldUI.java':[
144,
166,
],
'project:com/sun/jmx/mbeanserver/JmxMBeanServer.java':[
1196,
],
'project:com/sun/jmx/snmp/SnmpMessage.java':[
131,
136,
],
'project:com/sun/jmx/snmp/SnmpMsg.java':[
105,
],
'project:com/sun/jmx/snmp/SnmpOid.java':[
106,
498,
],
'project:com/sun/jmx/snmp/SnmpOidTableSupport.java':[
133,
],
'project:com/sun/jmx/snmp/SnmpV3Message.java':[
256,
],
'project:com/sun/jmx/snmp/agent/SnmpMibTable.java':[
547,
],
'project:com/sun/jmx/snmp/agent/SnmpTableSupport.java':[
360,
],
'project:com/sun/jmx/snmp/daemon/ClientHandler.java':[
89,
],
'project:com/sun/jmx/snmp/daemon/SnmpInformRequest.java':[
405,
],
'project:com/sun/jmx/snmp/daemon/SnmpQManager.java':[
194,
289,
],
'project:com/sun/jmx/snmp/daemon/SnmpSession.java':[
228,
399,
],
'project:com/sun/jmx/snmp/daemon/SnmpSocket.java':[
147,
],
'project:com/sun/jmx/snmp/defaults/DefaultPaths.java':[
225,
231,
],
'project:com/sun/jmx/snmp/tasks/ThreadService.java':[
183,
],
'project:com/sun/org/apache/bcel/internal/classfile/CodeException.java':[
228,
],
'project:com/sun/org/apache/bcel/internal/classfile/Constant.java':[
114,
],
'project:com/sun/org/apache/bcel/internal/classfile/ConstantPool.java':[
364,
],
'project:com/sun/org/apache/bcel/internal/classfile/FieldOrMethod.java':[
216,
],
'project:com/sun/org/apache/bcel/internal/classfile/InnerClass.java':[
220,
],
'project:com/sun/org/apache/bcel/internal/classfile/JavaClass.java':[
468,
642,
],
'project:com/sun/org/apache/bcel/internal/classfile/LineNumber.java':[
163,
],
'project:com/sun/org/apache/bcel/internal/classfile/LocalVariable.java':[
258,
],
'project:com/sun/org/apache/bcel/internal/classfile/StackMapEntry.java':[
185,
],
'project:com/sun/org/apache/bcel/internal/classfile/StackMapType.java':[
159,
],
'project:com/sun/org/apache/bcel/internal/generic/MethodGen.java':[
453,
518,
565,
],
'project:com/sun/org/apache/bcel/internal/util/Class2HTML.java':[
231,
],
'project:com/sun/org/apache/bcel/internal/util/ClassPath.java':[
224,
],
'project:com/sun/org/apache/xalan/internal/lib/ExsltDatetime.java':[
864,
906,
1018,
1034,
1052,
1064,
1076,
],
'project:com/sun/org/apache/xalan/internal/lib/ObjectFactory.java':[
340,
366,
601,
],
'project:com/sun/org/apache/xalan/internal/lib/SecuritySupport12.java':[
54,
67,
80,
],
'project:com/sun/org/apache/xalan/internal/xslt/ObjectFactory.java':[
340,
366,
603,
],
'project:com/sun/org/apache/xalan/internal/xslt/Process.java':[
720,
766,
899,
955,
971,
972,
999,
1000,
1005,
1032,
1048,
1049,
1115,
1204,
],
'project:com/sun/org/apache/xalan/internal/xslt/SecuritySupport12.java':[
54,
67,
80,
],
'project:com/sun/org/apache/xalan/internal/xsltc/cmdline/ObjectFactory.java':[
340,
366,
603,
],
'project:com/sun/org/apache/xalan/internal/xsltc/cmdline/SecuritySupport12.java':[
54,
67,
80,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/ObjectFactory.java':[
340,
366,
603,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/Parser.java':[
458,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/SecuritySupport12.java':[
54,
67,
80,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/Step.java':[
408,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/Whitespace.java':[
516,
518,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/util/ObjectFactory.java':[
340,
366,
603,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/util/ReferenceType.java':[
96,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/util/SecuritySupport12.java':[
54,
67,
80,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/ObjectFactory.java':[
340,
366,
603,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/SecuritySupport12.java':[
54,
67,
80,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/XSLTCDTMManager.java':[
440,
441,
],
'project:com/sun/org/apache/xalan/internal/xsltc/runtime/ObjectFactory.java':[
340,
366,
603,
],
'project:com/sun/org/apache/xalan/internal/xsltc/runtime/SecuritySupport12.java':[
54,
67,
80,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/ObjectFactory.java':[
340,
366,
603,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/SAX2DOM.java':[
155,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/SecuritySupport12.java':[
54,
67,
80,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/TrAXFilter.java':[
81,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/TransformerFactoryImpl.java':[
553,
819,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/TransformerImpl.java':[
733,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/Util.java':[
121,
],
'project:com/sun/org/apache/xerces/internal/dom/DOMConfigurationImpl.java':[
329,
671,
692,
725,
],
'project:com/sun/org/apache/xerces/internal/dom/DOMNormalizer.java':[
711,
712,
],
'project:com/sun/org/apache/xerces/internal/dom/ObjectFactory.java':[
223,
249,
492,
],
'project:com/sun/org/apache/xerces/internal/dom/SecuritySupport.java':[
57,
70,
83,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLDocumentScannerImpl.java':[
340,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLEntityManager.java':[
1765,
1972,
2106,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/XMLDTDValidator.java':[
690,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/ObjectFactory.java':[
223,
249,
492,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/SecuritySupport.java':[
57,
70,
83,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/xs/AnyURIDV.java':[
43,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/xs/XSSimpleTypeDecl.java':[
1934,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/SchemaGrammar.java':[
1417,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XMLSchemaLoader.java':[
1030,
1261,
1281,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XMLSchemaValidator.java':[
2440,
2483,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XSConstraints.java':[
1333,
1382,
1433,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/opti/SchemaParsingConfig.java':[
444,
449,
501,
506,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/traversers/XSDComplexTypeTraverser.java':[
456,
896,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/traversers/XSDElementTraverser.java':[
125,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/traversers/XSDHandler.java':[
681,
982,
1083,
2210,
2232,
2261,
2797,
2835,
2873,
2911,
2949,
3516,
3527,
3534,
3540,
3554,
3561,
],
'project:com/sun/org/apache/xerces/internal/jaxp/SchemaValidatorConfiguration.java':[
106,
],
'project:com/sun/org/apache/xerces/internal/jaxp/validation/ValidatorHandlerImpl.java':[
684,
],
'project:com/sun/org/apache/xerces/internal/parsers/AbstractDOMParser.java':[
723,
1365,
],
'project:com/sun/org/apache/xerces/internal/parsers/DOMParserImpl.java':[
256,
488,
506,
550,
597,
619,
633,
735,
767,
773,
],
'project:com/sun/org/apache/xerces/internal/parsers/ObjectFactory.java':[
221,
247,
490,
],
'project:com/sun/org/apache/xerces/internal/parsers/SecuritySupport.java':[
57,
70,
83,
],
'project:com/sun/org/apache/xerces/internal/parsers/XIncludeParserConfiguration.java':[
216,
],
'project:com/sun/org/apache/xerces/internal/util/DOMUtil.java':[
854,
869,
],
'project:com/sun/org/apache/xerces/internal/util/SecurityManager.java':[
189,
200,
212,
],
'project:com/sun/org/apache/xerces/internal/util/StAXInputSource.java':[
91,
],
'project:com/sun/org/apache/xerces/internal/util/XMLCatalogResolver.java':[
283,
353,
371,
],
'project:com/sun/org/apache/xerces/internal/xinclude/ObjectFactory.java':[
221,
247,
490,
],
'project:com/sun/org/apache/xerces/internal/xinclude/SecuritySupport.java':[
57,
70,
83,
],
'project:com/sun/org/apache/xerces/internal/xinclude/XIncludeHandler.java':[
417,
431,
570,
],
'project:com/sun/org/apache/xml/internal/dtm/DTMException.java':[
324,
],
'project:com/sun/org/apache/xml/internal/dtm/DTMManager.java':[
363,
],
'project:com/sun/org/apache/xml/internal/dtm/ObjectFactory.java':[
340,
366,
603,
],
'project:com/sun/org/apache/xml/internal/dtm/SecuritySupport12.java':[
54,
67,
80,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMDefaultBase.java':[
651,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMDocumentImpl.java':[
2059,
2061,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMManagerDefault.java':[
437,
438,
471,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/ObjectFactory.java':[
340,
366,
603,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/SecuritySupport12.java':[
54,
67,
80,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/sax2dtm/SAX2DTM2.java':[
483,
486,
],
'project:com/sun/org/apache/xml/internal/security/encryption/XMLCipher.java':[
1324,
],
'project:com/sun/org/apache/xml/internal/security/keys/keyresolver/implementations/EncryptedKeyResolver.java':[
119,
],
'project:com/sun/org/apache/xml/internal/security/transforms/implementations/TransformXSLT.java':[
70,
],
'project:com/sun/org/apache/xml/internal/security/utils/CachedXPathFuncHereAPI.java':[
384,
],
'project:com/sun/org/apache/xml/internal/security/utils/JavaUtils.java':[
100,
],
'project:com/sun/org/apache/xml/internal/security/utils/XMLUtils.java':[
166,
],
'project:com/sun/org/apache/xml/internal/security/utils/resolver/implementations/ResolverLocalFilesystem.java':[
142,
],
'project:com/sun/org/apache/xml/internal/serialize/HTMLdtd.java':[
416,
],
'project:com/sun/org/apache/xml/internal/serialize/ObjectFactory.java':[
221,
247,
490,
],
'project:com/sun/org/apache/xml/internal/serialize/OutputFormat.java':[
908,
932,
],
'project:com/sun/org/apache/xml/internal/serialize/SecuritySupport.java':[
57,
70,
83,
],
'project:com/sun/org/apache/xml/internal/serialize/SerializerFactory.java':[
76,
],
'project:com/sun/org/apache/xml/internal/serializer/CharInfo.java':[
183,
218,
305,
510,
516,
],
'project:com/sun/org/apache/xml/internal/serializer/Encodings.java':[
318,
],
'project:com/sun/org/apache/xml/internal/serializer/ObjectFactory.java':[
339,
365,
602,
],
'project:com/sun/org/apache/xml/internal/serializer/SecuritySupport12.java':[
51,
64,
77,
],
'project:com/sun/org/apache/xml/internal/serializer/SerializerBase.java':[
1064,
1261,
],
'project:com/sun/org/apache/xml/internal/serializer/ToStream.java':[
3100,
],
'project:com/sun/org/apache/xml/internal/serializer/ToXMLStream.java':[
497,
625,
],
'project:com/sun/org/apache/xml/internal/utils/DOMHelper.java':[
98,
],
'project:com/sun/org/apache/xml/internal/utils/FastStringBuffer.java':[
1148,
1164,
1182,
],
'project:com/sun/org/apache/xml/internal/utils/ObjectFactory.java':[
335,
361,
598,
],
'project:com/sun/org/apache/xml/internal/utils/ObjectPool.java':[
143,
144,
],
'project:com/sun/org/apache/xml/internal/utils/SecuritySupport12.java':[
54,
67,
80,
],
'project:com/sun/org/apache/xml/internal/utils/XMLReaderManager.java':[
138,
139,
],
'project:com/sun/org/apache/xpath/internal/SourceTreeManager.java':[
366,
368,
],
'project:com/sun/org/apache/xpath/internal/XPathContext.java':[
328,
530,
],
'project:com/sun/org/apache/xpath/internal/XPathException.java':[
188,
267,
],
'project:com/sun/org/apache/xpath/internal/compiler/SecuritySupport12.java':[
54,
67,
80,
],
'project:com/sun/org/apache/xpath/internal/functions/ObjectFactory.java':[
340,
366,
603,
],
'project:com/sun/org/apache/xpath/internal/functions/SecuritySupport12.java':[
54,
67,
80,
],
'project:com/sun/org/apache/xpath/internal/objects/XString.java':[
138,
],
'project:java/awt/AWTKeyStroke.java':[
242,
243,
],
'project:java/awt/Component.java':[
7945,
8583,
8584,
],
'project:java/awt/ComponentOrientation.java':[
184,
],
'project:java/awt/DefaultKeyboardFocusManager.java':[
113,
115,
117,
],
'project:java/awt/Font.java':[
1552,
1625,
],
'project:java/awt/Toolkit.java':[
1686,
2155,
2424,
],
'project:java/awt/Window.java':[
1330,
],
'project:java/awt/color/ICC_Profile.java':[
1096,
],
'project:java/awt/datatransfer/MimeType.java':[
348,
],
'project:java/awt/datatransfer/MimeTypeParameterList.java':[
304,
],
'project:java/awt/datatransfer/SystemFlavorMap.java':[
878,
899,
931,
],
'project:java/awt/dnd/SerializationTester.java':[
46,
],
'project:java/awt/font/NumericShaper.java':[
1379,
],
'project:java/awt/font/StyledParagraph.java':[
445,
463,
],
'project:java/awt/font/TextLayout.java':[
464,
],
'project:java/awt/font/TextLine.java':[
1301,
],
'project:java/awt/font/TransformAttribute.java':[
151,
],
'project:java/beans/DefaultPersistenceDelegate.java':[
208,
410,
],
'project:java/beans/Introspector.java':[
594,
],
'project:java/beans/MetaData.java':[
1337,
],
'project:java/beans/XMLEncoder.java':[
414,
700,
],
'project:java/io/Console.java':[
528,
562,
],
'project:java/io/FilterOutputStream.java':[
158,
],
'project:java/io/InputStream.java':[
185,
],
'project:java/io/LineNumberInputStream.java':[
156,
],
'project:java/io/ObjectInputStream.java':[
1270,
1275,
],
'project:java/io/ObjectOutputStream.java':[
1075,
1080,
],
'project:java/io/ObjectStreamClass.java':[
1572,
1602,
1645,
],
'project:java/lang/ApplicationShutdownHooks.java':[
107,
],
'project:java/lang/Boolean.java':[
239,
240,
],
'project:java/lang/CharacterName.java':[
91,
],
'project:java/lang/Integer.java':[
888,
889,
894,
],
'project:java/lang/Long.java':[
927,
928,
933,
],
'project:java/lang/Package.java':[
548,
],
'project:java/lang/StringCoding.java':[
188,
338,
],
'project:java/lang/Thread.java':[
1684,
1690,
],
'project:java/lang/UNIXProcess.java':[
234,
235,
236,
],
'project:java/lang/invoke/FilterGeneric.java':[
194,
200,
201,
207,
212,
213,
],
'project:java/lang/invoke/FromGeneric.java':[
211,
217,
218,
224,
229,
230,
],
'project:java/lang/invoke/MethodHandleImpl.java':[
239,
298,
1017,
1193,
],
'project:java/lang/invoke/MethodHandleNatives.java':[
350,
],
'project:java/lang/invoke/MethodHandleProxies.java':[
180,
],
'project:java/lang/invoke/SpreadGeneric.java':[
174,
180,
181,
189,
194,
195,
],
'project:java/lang/invoke/ToGeneric.java':[
291,
298,
299,
304,
309,
310,
],
'project:java/lang/ref/Finalizer.java':[
106,
],
'project:java/lang/ref/Reference.java':[
134,
],
'project:java/lang/reflect/Proxy.java':[
370,
],
'project:java/net/AbstractPlainSocketImpl.java':[
489,
616,
],
'project:java/net/CookieManager.java':[
373,
384,
],
'project:java/net/HttpURLConnection.java':[
514,
545,
],
'project:java/net/Inet6Address.java':[
239,
435,
],
'project:java/net/InetAddress.java':[
901,
1329,
],
'project:java/net/NetworkInterface.java':[
120,
163,
],
'project:java/net/Socket.java':[
676,
1504,
],
'project:java/net/SocketInputStream.java':[
172,
],
'project:java/net/SocketPermission.java':[
1049,
],
'project:java/net/URLConnection.java':[
619,
642,
666,
1256,
1333,
],
'project:java/nio/channels/spi/AbstractInterruptibleChannel.java':[
166,
],
'project:java/nio/file/FileSystems.java':[
387,
398,
],
'project:java/rmi/activation/Activatable.java':[
441,
],
'project:java/rmi/dgc/VMID.java':[
144,
],
'project:java/rmi/server/LogStream.java':[
251,
],
'project:java/security/cert/X509CertSelector.java':[
1925,
],
'project:java/text/DecimalFormat.java':[
2863,
],
'project:java/text/DecimalFormatSymbols.java':[
363,
586,
639,
],
'project:java/text/SimpleDateFormat.java':[
628,
1330,
1723,
2033,
],
'project:java/util/Calendar.java':[
2717,
],
'project:java/util/Date.java':[
285,
],
'project:java/util/Formatter.java':[
2954,
],
'project:java/util/Locale.java':[
1714,
1776,
],
'project:java/util/Properties.java':[
710,
],
'project:java/util/ResourceBundle.java':[
586,
587,
2576,
],
'project:java/util/Timer.java':[
556,
],
'project:java/util/TreeMap.java':[
196,
197,
316,
317,
639,
640,
2303,
2304,
],
'project:java/util/concurrent/AbstractExecutorService.java':[
244,
245,
290,
291,
],
'project:java/util/concurrent/ForkJoinPool.java':[
1025,
1028,
1096,
1121,
1303,
1323,
1364,
1378,
1653,
2117,
],
'project:java/util/concurrent/ForkJoinTask.java':[
540,
835,
],
'project:java/util/concurrent/ThreadPoolExecutor.java':[
426,
734,
770,
],
'project:java/util/jar/JarVerifier.java':[
365,
],
'project:java/util/jar/Manifest.java':[
271,
],
'project:java/util/prefs/AbstractPreferences.java':[
970,
],
'project:java/util/prefs/FileSystemPreferences.java':[
184,
],
'project:java/util/regex/Pattern.java':[
2638,
],
'project:java/util/regex/UnicodeProp.java':[
224,
],
'project:javax/imageio/stream/FileCacheImageInputStream.java':[
276,
],
'project:javax/imageio/stream/ImageInputStreamImpl.java':[
783,
875,
],
'project:javax/lang/model/SourceVersion.java':[
150,
],
'project:javax/naming/directory/BasicAttributes.java':[
275,
],
'project:javax/naming/spi/NamingManager.java':[
159,
],
'project:javax/print/PrintServiceLookup.java':[
188,
218,
360,
379,
390,
401,
441,
452,
],
'project:javax/print/StreamPrintServiceFactory.java':[
198,
],
'project:javax/security/sasl/Sasl.java':[
597,
],
'project:javax/sound/sampled/AudioSystem.java':[
224,
249,
250,
],
'project:javax/sql/rowset/RowSetMetaDataImpl.java':[
651,
718,
],
'project:javax/sql/rowset/serial/SQLOutputImpl.java':[
322,
],
'project:javax/sql/rowset/spi/SyncProviderException.java':[
134,
],
'project:javax/swing/AbstractButton.java':[
374,
],
'project:javax/swing/BufferStrategyPaintManager.java':[
213,
362,
478,
933,
],
'project:javax/swing/DebugGraphics.java':[
1302,
],
'project:javax/swing/DefaultDesktopManager.java':[
97,
110,
127,
139,
158,
240,
275,
],
'project:javax/swing/JDesktopPane.java':[
465,
],
'project:javax/swing/JEditorPane.java':[
594,
],
'project:javax/swing/JInternalFrame.java':[
1622,
1636,
1761,
1796,
],
'project:javax/swing/JOptionPane.java':[
1318,
1319,
1320,
1326,
1463,
1464,
1465,
1471,
1548,
1549,
1550,
1556,
1592,
2533,
],
'project:javax/swing/JTextArea.java':[
478,
],
'project:javax/swing/JTree.java':[
3170,
],
'project:javax/swing/TimerQueue.java':[
190,
],
'project:javax/swing/TransferHandler.java':[
1216,
],
'project:javax/swing/UIDefaults.java':[
194,
],
'project:javax/swing/colorchooser/AbstractColorChooserPanel.java':[
227,
],
'project:javax/swing/colorchooser/CenterLayout.java':[
76,
],
'project:javax/swing/colorchooser/ColorModel.java':[
97,
],
'project:javax/swing/colorchooser/ColorPanel.java':[
114,
],
'project:javax/swing/plaf/basic/BasicDesktopIconUI.java':[
183,
230,
],
'project:javax/swing/plaf/basic/BasicDesktopPaneUI.java':[
568,
572,
585,
601,
],
'project:javax/swing/plaf/basic/BasicHTML.java':[
63,
],
'project:javax/swing/plaf/basic/BasicInternalFrameTitlePane.java':[
630,
634,
638,
657,
659,
679,
683,
687,
],
'project:javax/swing/plaf/basic/BasicInternalFrameUI.java':[
643,
646,
648,
712,
],
'project:javax/swing/plaf/basic/BasicLookAndFeel.java':[
413,
2068,
2305,
],
'project:javax/swing/plaf/basic/BasicOptionPaneUI.java':[
825,
],
'project:javax/swing/plaf/basic/BasicSpinnerUI.java':[
692,
],
'project:javax/swing/plaf/basic/BasicTextUI.java':[
1177,
2513,
2514,
2515,
2581,
2582,
2591,
],
'project:javax/swing/plaf/basic/BasicToolBarUI.java':[
997,
1032,
],
'project:javax/swing/plaf/metal/MetalLookAndFeel.java':[
2226,
2227,
2228,
2229,
],
'project:javax/swing/plaf/metal/MetalUtils.java':[
180,
],
'project:javax/swing/plaf/nimbus/InternalFrameTitlePanePainter.java':[
71,
],
'project:javax/swing/plaf/nimbus/NimbusDefaults.java':[
1567,
1571,
],
'project:javax/swing/plaf/nimbus/OptionPaneMessageAreaPainter.java':[
71,
],
'project:javax/swing/plaf/nimbus/ScrollBarPainter.java':[
70,
],
'project:javax/swing/plaf/nimbus/SliderPainter.java':[
70,
],
'project:javax/swing/plaf/nimbus/SpinnerPainter.java':[
71,
],
'project:javax/swing/plaf/nimbus/SplitPanePainter.java':[
71,
],
'project:javax/swing/plaf/nimbus/TabbedPanePainter.java':[
75,
],
'project:javax/swing/plaf/synth/SynthContext.java':[
74,
75,
],
'project:javax/swing/plaf/synth/SynthDesktopIconUI.java':[
250,
],
'project:javax/swing/plaf/synth/SynthDesktopPaneUI.java':[
331,
343,
366,
375,
404,
],
'project:javax/swing/plaf/synth/SynthInternalFrameTitlePane.java':[
175,
],
'project:javax/swing/plaf/synth/SynthParser.java':[
584,
],
'project:javax/swing/text/AbstractDocument.java':[
1831,
],
'project:javax/swing/text/AbstractWriter.java':[
147,
],
'project:javax/swing/text/CompositeView.java':[
399,
400,
412,
413,
],
'project:javax/swing/text/DefaultCaret.java':[
1244,
1645,
],
'project:javax/swing/text/DefaultEditorKit.java':[
320,
1065,
1117,
1190,
1477,
1537,
1707,
],
'project:javax/swing/text/DefaultHighlighter.java':[
195,
212,
],
'project:javax/swing/text/DefaultStyledDocument.java':[
798,
],
'project:javax/swing/text/GlyphPainter2.java':[
131,
],
'project:javax/swing/text/InternationalFormatter.java':[
528,
529,
530,
1047,
1048,
],
'project:javax/swing/text/JTextComponent.java':[
2566,
2583,
2609,
2833,
3031,
3221,
3715,
3988,
4006,
4007,
4566,
4721,
4836,
4860,
4923,
],
'project:javax/swing/text/NumberFormatter.java':[
436,
],
'project:javax/swing/text/ParagraphView.java':[
287,
300,
],
'project:javax/swing/text/StyledEditorKit.java':[
579,
646,
712,
],
'project:javax/swing/text/html/AccessibleHTML.java':[
1059,
2829,
],
'project:javax/swing/text/html/CSS.java':[
1101,
1110,
2636,
2662,
2673,
2682,
2697,
2705,
],
'project:javax/swing/text/html/FormView.java':[
841,
904,
],
'project:javax/swing/text/html/HTMLDocument.java':[
2368,
2487,
3252,
3446,
3658,
3935,
],
'project:javax/swing/text/html/HTMLEditorKit.java':[
596,
819,
2001,
2113,
],
'project:javax/swing/text/html/IsindexView.java':[
110,
111,
],
'project:javax/swing/text/html/StyleSheet.java':[
308,
2112,
2143,
3163,
],
'project:javax/swing/text/html/TextAreaDocument.java':[
54,
65,
],
'project:javax/swing/text/html/parser/DTD.java':[
364,
],
'project:javax/swing/text/html/parser/DocumentParser.java':[
135,
],
'project:javax/xml/datatype/SecuritySupport.java':[
50,
],
'project:javax/xml/soap/FactoryFinder.java':[
94,
109,
134,
],
'project:javax/xml/transform/TransformerException.java':[
320,
],
'project:javax/xml/validation/SchemaFactoryFinder.java':[
447,
449,
451,
],
'project:javax/xml/ws/spi/FactoryFinder.java':[
113,
129,
140,
],
'project:javax/xml/xpath/SecuritySupport.java':[
51,
],
'project:javax/xml/xpath/XPathFactoryFinder.java':[
386,
388,
390,
],
'project:org/omg/CORBA/_IDLTypeStub.java':[
142,
151,
],
'project:org/omg/CORBA/_PolicyStub.java':[
153,
162,
],
'project:org/w3c/dom/bootstrap/DOMImplementationRegistry.java':[
362,
],
'project:org/xml/sax/helpers/XMLReaderFactory.java':[
167,
],
}
